1731597499681	Sync.LogManager	DEBUG	Flushing file log
1731597499682	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1731597499686	Sync.LogManager	DEBUG	Log cleanup threshold time: 1730733499686
1731597499704	Sync.LogManager	DEBUG	Done deleting files.
1731597532395	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1731597532396	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1731597532409	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1731597532429	Sync.SyncScheduler	DEBUG	Woke from sleep.
1731597532433	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1731597532435	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1731597532435	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1731597532436	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1731597534437	Sync.Service	DEBUG	User-Agent: Firefox/134.0a1 (Linux x86_64) FxSync/1.136.0.20241113094833.desktop
1731597534437	Sync.Service	INFO	Starting sync at 2024-11-14 15:18:54 in browser session qP6dtgeQpkt0
1731597534437	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1731597534441	Sync.Status	INFO	Resetting Status.
1731597534441	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1731597534442	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1731597534458	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/263878404/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1731597534459	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1731597534459	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1731597534459	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1731597534460	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1731597534463	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
